cernopendata_client/cli.py

```diff
@@ -225,6 +225,7 @@ def download_files(
       $ cernopendata-client download-files --recid 5500 --filter-regexp py --filter-range 1-2
       $ cernopendata-client download-files --recid 5500 --filter-regexp py --filter-range 1-2,3-4
     """
+    validate_server(server)
```

Comment:
Works nicely than before, but some output could be confusing to users:

```console
$ cernopendata-client download-files --recid 5500  --verify --server foo
==> ERROR: Server should be a valid URL
Usage: cernopendata-client download-files [OPTIONS]

Error: Invalid value for '--server': None
```

Firstly, where `None` comes from?  One would expect to see `foo` printed there.

Secondly, if somebody passes `https://foo.com`, the same output is produced. Here, the URL passed by the user is apparently valid... so we could rather say something like `HTTP URI` in the message, to exclude `HTTPS URI`, as it were...

Reply:
I am looking for how `Error: Invalid value for '--server': None` this is generated!

Reply:
Done 👍